timber-lint/src/main/java/timber/lint/WrongTimberUsageDetector.java

```diff
@@ -550,9 +552,13 @@ private boolean isLoggingExceptionMessage(JavaContext context, UExpression arg)
         "java.lang.Throwable");
   }
 
-  private static boolean isField(UExpression expression) {
-    return expression instanceof USimpleNameReferenceExpression
-        && (((USimpleNameReferenceExpression) expression).resolve() instanceof PsiField);
+  private static boolean canEvaluateExpression(UExpression expression) {
+    // TODO - try using CallGraph?
```

Comment:
Ooooo I went to an internal presentation about this. It's sweet.

Reply:
Yea, Tor mentioned it briefly at KotlinConf 2017.  I'm curious to learn what it can do.